fineract-provider/src/main/java/org/apache/fineract/accounting/closure/service/GLClosureWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -142,8 +144,7 @@ public CommandProcessingResult deleteGLClosure(final Long glClosureId) {
      * @param command
      * @param dve
      */
-    private void handleGLClosureIntegrityIssues(final JsonCommand command, final DataIntegrityViolationException dve) {
-        final Throwable realCause = dve.getMostSpecificCause();
+    private void handleGLClosureIntegrityIssues(final JsonCommand command, final Throwable realCause, final Exception dve) {
```

Comment:
@xurror it's not super important, but I thought perhaps this code review feedback could interest you: Instead of changing this like you have, what you really want to do in a case like this is just use the common super class of `JpaSystemException` and `DataIntegrityViolationException` like suggested below, does this make sense? Do you want to change this like that? It makes the code easier to read.

```suggestion
    private void handleGLClosureIntegrityIssues(final JsonCommand command, final NestedRuntimeException dve) {
        final Throwable realCause = dve.getMostSpecificCause();
```

Reply:
@vorburger instead NestedRuntimeException I used NonTransientDataAccessException which is a more suited super class.
You can check this and if it's LGTY you can merge.